CMakeLists.txt

```diff
@@ -1107,19 +1118,24 @@ if(ENABLE_HOST_CPU_DEVICES)
   set(BUILD_PTHREAD 1)
 endif()
 
-# The accel device could be built by default, but it's implemented in C++,
-# thus requires a C++ compiler, so let's not.
+# The accel device could be built by default, but it's implemented in C++, thus requires a C++ compiler, so let's not.
 if(ENABLE_ACCEL_DEVICE)
   set(BUILD_BASIC 1)
   set(BUILD_ACCEL 1)
   set(OCL_DRIVERS "${OCL_DRIVERS} accel")
 endif()
 
+# The tbb device is implemented in C++ and requires the Intel TBB library.
+if(ENABLE_TBB_DEVICE)
+  set(BUILD_BASIC 1)
+  set(BUILD_TBB 1)
```

Comment:
is this still needed?

Reply:
yes, ENABLE_* is the user-facing options while BUILD_* are internal options